sysl2/sysl/parse/listener_impl.go

```diff
@@ -1100,7 +1100,13 @@ func (s *TreeShapeListener) ExitDocumentation_stmts(*parser.Documentation_stmtsC
 
 // EnterQuery_var is called when production query_var is entered.
 func (s *TreeShapeListener) EnterQuery_var(ctx *parser.Query_varContext) {
-	var_name := ctx.Name().GetText()
+
```

Comment:
Doesn't the new linter complain about these leading empty lines?

Reply:
I think that file is excluded becuase it has a ton of bad lines already